lib/snorenotify.pri

```diff
@@ -0,0 +1,16 @@
+# Requires cmake doxygen extra-cmake-modules
```

Comment:
Why does it require doxygen?

Reply:
I haven't worked on this for a lot of weeks, it's leftover code that should be removed